src/block-components/image/index.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! function_exists( 'stackable_load_image_optimizer_polyfill_frontend_script' ) ) {
+	function stackable_load_image_optimizer_polyfill_frontend_script( $block_content, $block ) {
```

Comment:
Please add some comments here to explain why we need to load this file when EWWWW is present